examples/terraform/vmware-cloud-director/output.tf

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2022 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+output "kubeone_api" {
+  description = "kube-apiserver LB endpoint"
+
+  value = {
+    endpoint                    = vcd_vapp_vm.control_plane.0.network.0.ip
```

Comment:
Why are we taking the first control plane instead of bastion here?

Reply:
We'll remove the bastion, I forgot to remove the VM as well I guess.

Reply:
I have added that, good point 👍